.eslintrc.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
     'import/prefer-default-export': 'off',
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'warn',
+    'react/no-array-index-key': 'off',
```

Comment:
Please don't do this. These rules are set in place for a reason. I know why you did this, but this is not a solution to the problem you were facing. 

Head over the place where you were trying to index a key. I provided some suggestions.